steps/27/README.md

```diff
@@ -85,75 +85,109 @@ We create a new `unitTests.qunit.ts` file under `webapp/test/unit/`.
+### webapp/test/testsuite.qunit.html \(New\)
 
-First, we load some basic QUnit functionality via script tags. The QUnit test suite must be included at the end via a script tag which loads `unitTests.qunit.js`. The file extension `.js` must be used since this loads the transpiled version of `unitTests.qunit.ts`.
+We create a new `testsuite.qunit.html` page under `webapp/test`. 
+This page serves as an entry point for all tests within our project. For more information about the UI5 Test Suite, read [Concept and Basic Setup](https://sdk.openui5.org/#/topic/22f50c0f0b104bf3ba84620880793d3f).
 
 ```html
 <!DOCTYPE html>
 <html>
 <head>
-	<title>UI5 TypeScript Walkthrough - Unit Tests</title>
 	<meta charset="utf-8">
-
+	<title>QUnit test suite for UI5 TypeScript Walkthrough</title>
 	<script
-		id="sap-ui-bootstrap"
-		src="../../resources/sap-ui-core.js"
-		data-sap-ui-resourceroots='{
-			"ui5.walkthrough": "../../"
+		src="../resources/sap/ui/test/starter/createSuite.js"
+		data-sap-ui-testsuite="test-resources/ui5/walkthrough/testsuite.qunit"
+		data-sap-ui-resource-roots='{
+			"test-resources.ui5.walkthrough": "./"
 		}'
-		data-sap-ui-compat-version="edge"
-		data-sap-ui-async="true">
-	</script>
-
-	<link rel="stylesheet" type="text/css" href="../../resources/sap/ui/thirdparty/qunit-2.css">
-
-	<script src="../../resources/sap/ui/thirdparty/qunit-2.js"></script>
-	<script src="../../resources/sap/ui/qunit/qunit-junit.js"></script>
-
-	<script src="./unitTests.qunit.js"></script>
+	></script>
 </head>
 <body>
-	<div id="qunit"/>
-	<div id="qunit-fixture"/>
 </body>
 </html>
-
 ```
 
-The so-called QUnit test suite is an HTML page that triggers all QUnit tests for the application. Most of it is generating the layout of the result page that you can see in the preview and we won’t further explain these parts.
+***
+
+### webapp/test/testsuite.qunit.ts \(New\)
 
-If we now open the `webapp/test/unit/unitTests.qunit.html` file in the browser, we should see our test running and verifying the formatter logic.
+Now, we create the corresponding configuration next to the test suite itself.
+
+```ts
+export default {
+	name: "QUnit test suite for UI5 TypeScript Walkthrough",
+	defaults: {
+		page: "ui5://test-resources/ui5/walkthrough/Test.qunit.html?testsuite={suite}&test={name}",
+		qunit: {
+			version: 2
+		},
+		sinon: {
+			version: 1
+		},
+		ui5: {
+			language: "EN",
+			theme: "sap_horizon"
+		},
+		coverage: {
+			only: "ui5/walkthrough/",
+			never: [
+				"ui5/walkthrough/test/",
+				"ui5/walkthrough/localService/"
+			]
+		},
+		loader: {
+			paths: {
+				"ui5/walkthrough": "../"
+			}
+		}
+	},
+	tests: {
+		"unit/unitTests": {
+			title: "UI5 TypeScript Walkthrough - Unit Tests"
+		}
+	}
+};
+```
 
 ***
 
+### webapp/test/Test.qunit.html \(New\)
+
+Finally, we create a generic test page. It will be called with the test suite and test name in order to run a test.
+
```

Comment:
I guess it would be good to at least explain src="../resources/sap/ui/test/starter/runTest.js"